item.go

```diff
@@ -208,6 +208,13 @@ func (client *Client) GetItems(appId int64) (items *ItemList, err error) {
 	return
 }
 
+// https://developers.podio.com/doc/items/filter-items-4496747
+func (client *Client) filterItems(appId int64, params map[string]interface{}) (items *ItemList, err error) {
```

Comment:
I guess the intention is to export this function? If so, then the name should be capitalised :)